presto-spi/src/main/java/io/prestosql/spi/StandardErrorCode.java

```diff
@@ -20,129 +20,129 @@
-    OPTIMIZER_TIMEOUT(0x0001_0014, INTERNAL_ERROR),
-    OUT_OF_SPILL_SPACE(0x0001_0015, INTERNAL_ERROR),
-    REMOTE_HOST_GONE(0x0001_0016, INTERNAL_ERROR),
-    CONFIGURATION_INVALID(0x0001_0017, INTERNAL_ERROR),
-    CONFIGURATION_UNAVAILABLE(0x0001_0018, INTERNAL_ERROR),
-    INVALID_RESOURCE_GROUP(0x0001_0019, INTERNAL_ERROR),
+    GENERIC_INTERNAL_ERROR(65536, INTERNAL_ERROR),
+    TOO_MANY_REQUESTS_FAILED(65537, INTERNAL_ERROR),
+    PAGE_TOO_LARGE(65538, INTERNAL_ERROR),
+    PAGE_TRANSPORT_ERROR(65539, INTERNAL_ERROR),
+    PAGE_TRANSPORT_TIMEOUT(65540, INTERNAL_ERROR),
+    NO_NODES_AVAILABLE(65541, INTERNAL_ERROR),
+    REMOTE_TASK_ERROR(65542, INTERNAL_ERROR),
+    COMPILER_ERROR(65543, INTERNAL_ERROR),
+    REMOTE_TASK_MISMATCH(65544, INTERNAL_ERROR),
+    SERVER_SHUTTING_DOWN(65545, INTERNAL_ERROR),
+    FUNCTION_IMPLEMENTATION_MISSING(65546, INTERNAL_ERROR),
+    REMOTE_BUFFER_CLOSE_FAILED(65547, INTERNAL_ERROR),
+    SERVER_STARTING_UP(65548, INTERNAL_ERROR),
+    FUNCTION_IMPLEMENTATION_ERROR(65549, INTERNAL_ERROR),
+    INVALID_PROCEDURE_DEFINITION(65550, INTERNAL_ERROR),
+    PROCEDURE_CALL_FAILED(65551, INTERNAL_ERROR),
+    AMBIGUOUS_FUNCTION_IMPLEMENTATION(65552, INTERNAL_ERROR),
+    ABANDONED_TASK(65553, INTERNAL_ERROR),
+    CORRUPT_SERIALIZED_IDENTITY(65554, INTERNAL_ERROR),
+    CORRUPT_PAGE(65555, INTERNAL_ERROR),
+    OPTIMIZER_TIMEOUT(65556, INTERNAL_ERROR),
+    OUT_OF_SPILL_SPACE(65557, INTERNAL_ERROR),
+    REMOTE_HOST_GONE(65558, INTERNAL_ERROR),
+    CONFIGURATION_INVALID(65559, INTERNAL_ERROR),
+    CONFIGURATION_UNAVAILABLE(65560, INTERNAL_ERROR),
+    INVALID_RESOURCE_GROUP(65561, INTERNAL_ERROR),
 
-    GENERIC_INSUFFICIENT_RESOURCES(0x0002_0000, INSUFFICIENT_RESOURCES),
-    EXCEEDED_GLOBAL_MEMORY_LIMIT(0x0002_0001, INSUFFICIENT_RESOURCES),
-    QUERY_QUEUE_FULL(0x0002_0002, INSUFFICIENT_RESOURCES),
-    EXCEEDED_TIME_LIMIT(0x0002_0003, INSUFFICIENT_RESOURCES),
-    CLUSTER_OUT_OF_MEMORY(0x0002_0004, INSUFFICIENT_RESOURCES),
-    EXCEEDED_CPU_LIMIT(0x0002_0005, INSUFFICIENT_RESOURCES),
-    EXCEEDED_SPILL_LIMIT(0x0002_0006, INSUFFICIENT_RESOURCES),
-    EXCEEDED_LOCAL_MEMORY_LIMIT(0x0002_0007, INSUFFICIENT_RESOURCES),
-    ADMINISTRATIVELY_PREEMPTED(0x0002_0008, INSUFFICIENT_RESOURCES),
+    GENERIC_INSUFFICIENT_RESOURCES(131072, INSUFFICIENT_RESOURCES),
+    EXCEEDED_GLOBAL_MEMORY_LIMIT(131073, INSUFFICIENT_RESOURCES),
+    QUERY_QUEUE_FULL(131074, INSUFFICIENT_RESOURCES),
+    EXCEEDED_TIME_LIMIT(131075, INSUFFICIENT_RESOURCES),
+    CLUSTER_OUT_OF_MEMORY(131076, INSUFFICIENT_RESOURCES),
+    EXCEEDED_CPU_LIMIT(131077, INSUFFICIENT_RESOURCES),
+    EXCEEDED_SPILL_LIMIT(131078, INSUFFICIENT_RESOURCES),
+    EXCEEDED_LOCAL_MEMORY_LIMIT(131079, INSUFFICIENT_RESOURCES),
+    ADMINISTRATIVELY_PREEMPTED(131080, INSUFFICIENT_RESOURCES),
     /**/;
 
-    // Connectors can use error codes starting at the range 0x0100_0000
+    // Connectors can use error codes starting at 16777216
```

Comment:
Keep this as hex, since we do assignments to connectors using 2^16 hex blocks (see wiki page)